Formula/wine.rb

```diff
@@ -35,16 +35,9 @@ class Wine < Formula
   end
 
   devel do
-    url "https://dl.winehq.org/wine/source/2.x/wine-2.13.tar.xz"
-    mirror "https://downloads.sourceforge.net/project/wine/Source/wine-2.13.tar.xz"
-    sha256 "6fbca3ddd19b061ce65a267639efae93cf38cfabbe4a9c148a17a44703587ef8"
-
-    # Patch to fix screen-flickering issues. Still relevant on 2.3.
```

Comment:
Then it needs to be fixed and submitted to [formula-patches](https://github.com/Homebrew/formula-patches).

The part that fails to apply is

```
bash-3.2$ cat dlls/wined3d/context.c.rej
***************
*** 1946,1952 ****
      }
  
      /* Try to find a pixel format which matches our requirements. */
-     if (!(pixel_format = context_choose_pixel_format(device, hdc, color_format, ds_format, auxBuffers)))
          goto out;
  
      ret->gl_info = gl_info;
--- 1954,1960 ----
      }
  
      /* Try to find a pixel format which matches our requirements. */
+     if (!(pixel_format = context_choose_pixel_format(device, swapchain, hdc, color_format, ds_format, auxBuffers)))
          goto out;
  
      ret->gl_info = gl_info;
```

because that line has changed to

```
    /* Try to find a pixel format which matches our requirements. */
    if (!(ret->pixel_format = context_choose_pixel_format(device, ret->hdc, color_format, ds_format, auxBuffers)))
        goto out;
```